### PR TITLE
[docs-only] oCIS 1.20.0 release notes

### PR DESCRIPTION
**Pull request by pmaier1**
review plzzz

### CODE REVIEW

File: docs/ocis/release_notes.md [hunk 1]
```diff
@@ -7,6 +7,44 @@ geekdocEditPath: edit/master/docs/ocis
...
+* All Space members can now list all links and shares. [ocis#3370](https://github.com/owncloud/ocis/issues/3370)
+* The LDAP configuration settings have been simplified and unified across services. [#3476](https://github.com/owncloud/ocis/pull/3476)
+* All sharing options (users & links) are now united in one panel in ownCloud Web. [web#6701](https://github.com/owncloud/web/pull/6701)
+* The "Media Viewer" in ownCloud Web has been renamed to "Preview". [web#6514](https://github.com/owncloud/web/pull/6514)
+* ownCloud Web now has support for audio playback in "Preview" (e.g., MP3, WAV, FLAC, OGG). [web#6514](https://github.com/owncloud/web/pull/6514)
+* Public links on single files can now be created with the Editor role. [#6618](https://github.com/owncloud/web/pull/6618)
+* ownCloud Web can now download multiple files and folders as archives when used with the ownCloud 10 backend. [web#6239](https://github.com/owncloud/web/issues/6239)
```
Comment (micbar) [inline docs/ocis/release_notes.md:24]: Remove this?

Comment (micbar) [inline docs/ocis/release_notes.md:24]:
```suggestion
```

Comment (micbar) [reply docs/ocis/release_notes.md:24]: not ocis related

**Review by micbar — APPROVED** [review]
One line could be removed. You can commit my suggestion if you like

File: docs/ocis/release_notes.md (hunk 1 above)
Comment (pmaier1) [reply docs/ocis/release_notes.md:24]: I'd like to provide highlights for oC Web here as well and this definitely is one, IMO.

File: docs/ocis/release_notes.md [hunk 2]
```diff
@@ -7,6 +7,44 @@ geekdocEditPath: edit/master/docs/ocis
 geekdocFilePath: release_notes.md
 ---
 
+## ownCloud Infinite Scale 1.20.0 Technology Preview
+
+Version 1.20.0 brings major improvements, new features and bug fixes to the platform. Infinite Scale now provides complete Auditing capabilities and the basic Spaces' feature has reached initial feature completeness. Furthermore, ownCloud Web introduces a number of smaller features as well as more design and usability improvements.
```
Comment (kulmann) [inline docs/ocis/release_notes.md:12]:
```suggestion
Version 1.20.0 brings major improvements, new features and bug fixes to the platform. Infinite Scale now provides complete Auditing capabilities and the basic 'Spaces' feature has reached initial feature completeness. Furthermore, ownCloud Web introduces a number of smaller features as well as more design and usability improvements.
```

File: docs/ocis/release_notes.md [hunk 3]
```diff
@@ -7,6 +7,44 @@ geekdocEditPath: edit/master/docs/ocis
...
+* The implementation of the basic 'Spaces' feature is now complete. [web#6693](https://github.com/owncloud/web/pull/6693) [web#6659](https://github.com/owncloud/web/pull/6659) [web#6639](https://github.com/owncloud/web/pull/6639) [web#6633](https://github.com/owncloud/web/pull/6633) [web#6662](https://github.com/owncloud/web/pull/6662) [web#6642](https://github.com/owncloud/web/pull/6642)
+* All Space members can now list all links and shares. [ocis#3370](https://github.com/owncloud/ocis/issues/3370)
+* The LDAP configuration settings have been simplified and unified across services. [#3476](https://github.com/owncloud/ocis/pull/3476)
+* All sharing options (users & links) are now united in one panel in ownCloud Web. [web#6701](https://github.com/owncloud/web/pull/6701)
+* The "Media Viewer" in ownCloud Web has been renamed to "Preview". [web#6514](https://github.com/owncloud/web/pull/6514)
+* ownCloud Web now has support for audio playback in "Preview" (e.g., MP3, WAV, FLAC, OGG). [web#6514](https://github.com/owncloud/web/pull/6514)
+* Public links on single files can now be created with the Editor role. [#6618](https://github.com/owncloud/web/pull/6618)
```
Comment (kulmann) [inline docs/ocis/release_notes.md:23]: Did you double check if this really works with `vanilla oCIS`? afair this was EOS related...

File: docs/ocis/release_notes.md (hunk 1 above)
Comment (micbar) [reply docs/ocis/release_notes.md:24]: yes. But these are the ocis release notes. We cannot use this version of web with oc10.

There is also a changelog for web in the marketplace, where this highlight belongs to.

File: docs/ocis/release_notes.md (hunk 3 above)
Comment (micbar) [reply docs/ocis/release_notes.md:23]: Let me check tomorrow. I have seen an ocis backend ticket for that.

Comment (micbar) [reply docs/ocis/release_notes.md:23]: So, finally the feature is not part of vanilla oCIS, but will be switched on in the next release

Comment (micbar) [reply docs/ocis/release_notes.md:23]:
```suggestion
```

**Comment by pmaier1** [comment]
Thanks for the reviews. Ready to merge, I think.

**Pull request merged**